index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Landing Page Dashboard</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="main">
+        <div class="header-main">
+            <div class="header-container">
+                <div class="header">
+                    <div class="header-logo">Header Logo</div>
+                    <div class="header-links">
+                        <a>Header link 1</a>
+                        <a>Header link 2</a>
+                        <a>Header link 3</a>
+                    </div>
+                </div>
+                <div class="header-main-section">
+                    <div class="header-description"> 
+                        <div class="hero-main-text">This website is awesome</div>
+                        <div class="hero-secondary-text">
+                            This website has more subtext that goes under the main title. It's a smaller font and the color is lower contrast
+                        </div>
+                        <button class="sign-up-btn">Sign Up</button>
+                    </div>
+                    <img src="./images/header-image.jpg" alt="Shot of buildings"/>
+                </div>
+            </div>
+            <div class="info-container">
+                <div class="info-heading">
+                    Some Random Information
+                </div>
+                <div class="info-images">
+                    <figure class="item">
+                        <img src=""/>
```

Comment:
why you are using <img> tag with no image path ? also please add alt tag ?
@abhishek310001 Can you also attache UI screenshot in your PR please ?